go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/tidepool-org/go-common
 
-go 1.14
+go 1.18
```

Comment:
Why not `1.21`?

Reply:
My understanding of that value, is that it represents the minimum go toolchain version that is required to build/use the module. [Reference](https://go.dev/ref/mod#go-mod-file-go)

So 1.18 felt the most honest in that respect.